test/phpunit.xml.dist

```diff
@@ -1,33 +0,0 @@
-<phpunit 
```

Comment:
Didn't even knew about this feature - my main intention was to split up configs.
File can be safely renamed back to .dist, hovewer.